metric.py

```diff
@@ -228,9 +237,9 @@ def save(self, imgFolder, save2disk=False):
 def distance(X, Y, sqrt):
     nX = X.size(0)
     nY = Y.size(0)
-    X = X.view(nX,-1).cuda()
+    X = X.view(nX,-1)
     X2 = (X*X).sum(1).resize_(nX,1)
-    Y = Y.view(nY,-1).cuda()
+    Y = Y.view(nY,-1)

```

Comment:
This is a good catch! We only used the `--cuda` flag in the demo but not here. I think we should make it consistent. Maybe solve it in later commits.

Reply:
I got what's your idea. But in practice, the operation here takes huge RAM memory. On my `gtx970 4G`, the networks before could run smoothly, but i can't run these operations here even when `sampleSize` is 500!     
Besides, the operations here just takes seconds on CPU, so there is no need to put the tensors into `cuda`.

Reply:
And I didn't rewrite your core codes like networks or metrics.    
I just modify the structure and fix some bugs which have no effects to the demo experiment outputs.

Reply:
Yea, I agree. Let's go with it for now!

Reply:
Done！